README.md

```diff
@@ -0,0 +1,38 @@
+# ASP.NET Core Single Page App templates
+
+This repo is home for the ASP.NET Core Single Page Application (SPA) project templates that ship as part of the .NET SDK. Currently, the .NET SDK includes template for Angular and React.
+
+Besides these two templates, we also want this repo to serve as a knowledge-base for all community-owned SPA project templates out there.
+We expect to maintain that list in the bottom of this file. If you know of any such project templates, which will meet some basic quality bar, submit a PR and we'll consider it. Things that we'll be looking for are:
+- HTTPS enabled by default
+- good dev experience - easy to understand and work with out of the box
+- publishing newly created project works
+- JS framework code kept up-to-date
+
+We try to keep the list of community owned SPA project templates up to date as well as want to make sure it is a good quality list for everyone to refer to. For that, submissions will be evaluated by our team on case-by-case basis.
+
+## How to propose a new SPA project template reference
+
+We've created a new PR template, to help with new project template proposals.
+The format of each project template in the file should follow existing pattern, which goes as follows:
+
+```
+1. **Template name** / [GitHub](https://github.com/**owner/repo**) /  [NuGet](https://www.nuget.org/packages/**packageName**/)<br />
+  Short description of the template
+```
+
+Simply copy the above template to the bottom of this file, and then modify it as necessary.
+
+When you're ready to propose your changes prepared in your fork, use the `/.github/PULL_REQUEST_TEMPLATE/projectTemplate.md` PR template when submitting a new PR. To do so, append the additional `&template=projectTemplate.md` parameter to the PR creation page link, and you'll get all the info we need to fill in for the PR.
+
```

Comment:
```suggestion
We try to keep the list of community owned SPA project templates up to date as well as want to make sure it is a good quality list for everyone to refer to. For that, submissions will be evaluated by our team on case-by-case basis.

## How to propose a new SPA project template reference

We've created a new PR template, to help with new project template proposals.
The format of each project template in the file should follow existing pattern, which goes as follows:

```
1. **Template name** / [GitHub](https://github.com/**owner/repo**) /  [NuGet](https://www.nuget.org/packages/**packageName**/)<br />
  Short description of the template
```

Simply copy the above template to the bottom of this file, and then modify it as necessary.

When you're ready to propose your changes prepared in your fork, use the `/.github/PULL_REQUEST_TEMPLATE/projectTemplate.md` PR template when submitting a new PR. To do so, append the additional `&template=projectTemplate.md` parameter to the PR creation page link, and you'll get all the info we need to fill in for the PR.

```